pkg/controller/replicaset/replica_set.go

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 // For RC, the objects are converted on the way in and out (see ../replication/),
 // as if ReplicationController were just an older API version of ReplicaSet.
 // However, RC and RS still have separate storage and separate instantiations
-// of the ReplicaSetController object.
+// of the Controller object.
```

Comment:
I'm not a big fan of blindly following golint errors, I'd prefer we don't change these.